src/sentry/api/endpoints/organization_check_has_mobile_app_events.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+
+from django.core.cache import cache
+from datetime import timedelta
+from django.utils import timezone
+
+from sentry.api.bases import OrganizationEventsEndpointBase
+from sentry.snuba import discover
+
+# one day cache
+CACHE_TTL = 24 * 60 * 60
+
+
+logger = logging.getLogger(__name__)
+
+
+class OrganizationCheckHasMobileAppEvents(OrganizationEventsEndpointBase):
+    def get(self, request, organization):
+        # cache is unique to an org
+        cache_key = f"check_mobile_app_events.{organization.id}"
```

Comment:
nit: Don't we usually separate cache keys with `:`
```suggestion
        cache_key = f"check_mobile_app_events:{organization.id}"
```